pyramid_oereb/standard/pyramid_oereb.yml.mako

```diff
@@ -64,9 +64,10 @@ pyramid_oereb:
     # The pyramid renderer which is used as proxy pass through to the desired service for printable static
     # extract. Here you can define the path to the logic which prepares the output as payload for print
     # service and returns the result to the user.
-    renderer: pyramid_oereb.contrib.print_proxy.mapfish_print.Renderer
-    # The minimum buffer in pixel at 72 DPI between the real estate and the map's border.
-    buffer: 30
+    # renderer: pyramid_oereb.contrib.print_proxy.mapfish_print.Renderer
+    renderer: pyramid_oereb.contrib.print_proxy.xml_2_pdf.Renderer
```

Comment:
Can we please change this to a variable instead?
Something like 
render: ${print_renderer}

Reply:
Feel free to do it.

Reply:
Exactly, that's why this should be a variable. But I can do that in a follow-up PR, to avoid mixing too many things.